pr-data.csv

```diff
@@ -2821,14 +2821,14 @@ https://github.com/apache/shardingsphere-elasticjob,b022898ef1b8c984e17efb2a422e
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.integrate.enable.OneOffEnabledJobIntegrateTest.assertJobInit,OD,DeveloperFixed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78;https://github.com/apache/shardingsphere-elasticjob/commit/054571c6d7caca455418e0f83056a0182315bd75
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.integrate.enable.ScheduleEnabledJobIntegrateTest.assertJobInit,OD,DeveloperFixed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78;https://github.com/apache/shardingsphere-elasticjob/commit/054571c6d7caca455418e0f83056a0182315bd75
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertListenException,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertMonitorWithDumpCommand,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertPortInvalid,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceEnableTest.assertMonitorWithCommand,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
+https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.integrate.disable.OneOffDisabledJobIntegrateTest.assertJobRunning,OD,DeveloperFixed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78;https://github.com/TestingResearchIllinois/idoft/issues/1012;https://github.com/apache/shardingsphere-elasticjob/commit/054571c6d7caca455418e0f83056a0182315bd75
```

Comment:
The `Notes` includes https://github.com/TestingResearchIllinois/idoft/issues/1012 which describes where the test was moved or renamed, but this line claims `DeveloperFixed`. Do you have some logs for `DeveloperFixed`?

Reply:
Updated the issues and also the PR description with log file path to my VM. Thanks!